examples/papers/flow_upscaling/example1/data.py

```diff
@@ -50,13 +50,11 @@ def flow(gb, model, data):
         d[pp.DISCRETIZATION_MATRICES] = {model_data: {}}
 
     for e, d in gb.edges():
-        g_l = gb.nodes_of_edge(e)[0]
         mg = d["mortar_grid"]
         check_P = mg.slave_to_mortar_avg()
-
-        aperture = gb.node_props(g_l, pp.PARAMETERS)[model_data]["aperture"]
-        gamma = check_P * np.power(aperture, 1 / (2.0 - g.dim))
-        kn = data["kf"] / mu * np.ones(mg.num_cells) / gamma
+        aperture = data["aperture"]
+        gamma = check_P * aperture
```

Comment:
gamma does not involve the power term, is this correct?

Reply:
I don't know where the name "gamma" comes from. But the aperture which the normal permeability should be scaled with has dimensions of length, and should not have be raised to the power (dim_max - dim), no.